iothub_client/inc/iothub_client_options.h

```diff
@@ -76,6 +76,8 @@ extern "C"
     //diagnostic sampling percentage value, [0-100]
     static STATIC_VAR_UNUSED const char* OPTION_DIAGNOSTIC_SAMPLING_PERCENTAGE = "diag_sampling_percentage";
 
+static STATIC_VAR_UNUSED const char* OPTION_CONVENIENCE_LOOP_TIME = "convenience_loop_timeout";
```

Comment:
I would suggest using another name.
Something like OPTION_DO_WORK_FREQUENCY_IN_MS.
I haven't heard the term CONVENIENCE_LOOP_TIME before when referring to the DOWork frequency, so it might be a little foreign to the customers if introduced as such.

Reply:
You should send an email to the team asking their opinion I think.

Reply:
After team consensus I've updated the names to center around OPTION_DO_WORK_FREQUENCY_IN_MS